src/dlgConnectionProfiles.cpp

```diff
@@ -716,166 +716,311 @@ QPair<bool, QString> dlgConnectionProfiles::writeProfileData(const QString& prof
-                             "find content you missed the first time around.\n\nThe Cleft opened in July 2000 and has been in active development ever since. We're always innovating. Recent features "
-                             "include Discord integration (https://discord.gg/cSqkpbu) and areas written with artificial intelligence. Check us out!");
-    } else if (hostUrl == QLatin1String("godwars2.org")) {
-        return QLatin1String(
-                "God Wars II is a fast and furious combat mud, designed to test player skill in terms of pre-battle preparation and on-the-spot reflexes, as well as the ability to adapt quickly to "
-                "new situations. Take on the role of a godlike supernatural being in a fight for supremacy.\n\nRoomless world. Manual combat. Endless possibilities.");
-    } else if (hostUrl == QLatin1String("3k.org")) {
+        // clang-format off
+        // wrap before the last word on a line is here so that the source text
+        // is no more than 80 characters excluding the indent for readbility in
+        // an editor. Note that the text is autowrapped in the widget used to
+        // show it so ensure lines end in a space (for locales that use spaces)
+        // and use two '\n' to get a blank line between paragraphs.                          ------>|
+        // clang-format on
+        return qsl("The Realms of Despair is the original SMAUG MUD and is FREE to play. We have an "
+                   "active Roleplaying community, an active player-killing (deadly) community, and "
+                   "a very active peaceful community. Players can choose from 13 classes (including "
+                   "a deadly-only class) and 13 races. Character appearances are customizable on "
+                   "creation and we have a vast collection of equipment that is level, gender, "
+                   "class, race and alignment specific. We boast well over 150 original, exclusive "
+                   "areas, with a total of over 20,000 rooms. Mob killing, or 'running' is one of "
+                   "our most popular activities, with monster difficulties varying from easy one-"
+                   "player kills to difficult group kills. We have four deadly-only Clans, twelve "
+                   "peaceful-only Guilds, eight Orders, and fourteen Role-playing Nations that "
+                   "players can join to interact more closely with other players. We have two mortal "
+                   "councils that actively work toward helping players: The Symposium hears ideas "
+                   "for changes, and the Newbie Council assists new players. Our team of Immortals "
+                   "are always willing to answer questions and to help out however necessary. Best "
+                   "of all, playing the Realms of Despair is totally FREE!");
+    }
+    if (hostUrl == QLatin1String("zombiemud.org")) {
```

Comment:
I like "else" better here, as this somewhat contradicts the previous and needn't be checked if that was true.

Reply:
Oh wait, if the "if" block actually ended in "return", then we can indeed drop the "else" here.

Reply:
This PR removes the "else-after-return" anti-patterns in this method that some *linters* - such as `clang-tidy` moan about, see: https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html . If the previous if test is true, execution never reaches here as the return short-circuits the remainder of the method.